### DISCUSSION TITLE
AutoTool - GPT3-powered text-to-dashboard with ToolJet

### DISCUSSION THREAD

**Discussion by jayhack** [ToolJet/ToolJet#4683] [Show and tell]
Hey all!

I made a conversational UI for ToolJet. It uses GPT-3 to generate the JSON for dashboards, then lets you iteratively modify them using more natural language commands.

How can we make this a plugin for ToolJet, instead of an external service?

https://twitter.com/mathemagic1an/status/1587443727735214080

Let me know what you think. Thanks!

**Comment by withshubh** [comment]
Hey @jayhack 👋 
AutoTool looks really great, Thanks for sharing this. 🙌 
Currently, We do not have a mechanism to build such plugins right now. We will think and get back to you in some time.

**Comment by etewiah** [comment]
Wait, is AutoTool an actual product or just vapourware?
